index.html

```diff
@@ -78,7 +78,15 @@
 var channel = 'bad-attitude';
 
 function clean(text) {
-  return (''+text).trim().replace( /[<>]/g, '' );
+  var i = 0, m;
```

Comment:
Doing this in clean() will apply it to nicks as well as text.  Is that a desired property?  I'm not totally sold.